Update when-then-rules.md (#34)
openremote/Documentation: docs/user-guide/rules-and-forecasting/when-then-rules.md

The remainder of the guide assumes you are using the [Demo](https://openremote.io/demo/) where all assets and attributes have already been created and configured:

> 1. Switch from the `master` realm to the `Smart city` realm in the top. You will see the assets used a demo setup. 

Reviewer: ```suggestion
1. Switch from the `master` realm to the `Smart city` realm in the top. You will see the assets used in the demo setup. 
```

2. Navigate to the `Rules` page, here you will find some demo rules that are running at the moment. We will add our own.
3. Add a new rule:
   * Click the `+`.
   * Select 'When-Then'.
   * Name it `Cold and sunset: lights on`.
4. Create the When side of the rule:
   * Click the `+`, a list of asset types will appear.

>    * Select `Weather Asset`, the next condition will appear showing `Any of this type`.
>    * Switch to the asset named `Weather`.

Reviewer: ```suggestion
   * Select `Weather Asset`, a condition will get added for this asset type.
   * Switch the asset from `Any of this type` to `Weather`.
```
